docs/apache-airflow-providers-google/operators/cloud/dataplex.rst

```diff
@@ -679,3 +679,115 @@ use :class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCatalogUp
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_dataplex_catalog_create_entry]
+    :end-before: [END howto_operator_dataplex_catalog_create_entry]
+
+.. _howto/operator:DataplexCatalogDeleteEntryOperator:
+
+Delete an Entry
+---------------
+
+To delete an Entry in specific location in Dataplex Catalog you can
+use :class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCatalogDeleteEntryOperator`
+
+.. exampleinclude:: /../../providers/tests/system/google/cloud/dataplex/example_dataplex_catalog.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_dataplex_catalog_delete_entry]
+    :end-before: [END howto_operator_dataplex_catalog_delete_delete]
+
+.. _howto/operator:DataplexCatalogListEntriesOperator:
+
+List Entries
+------------
+
+To list all Entries in specific location in Dataplex Catalog you can
+use :class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCatalogListEntriesOperator`.
+This operator also supports filtering and ordering the result of the operation.
+
+.. exampleinclude:: /../../providers/tests/system/google/cloud/dataplex/example_dataplex_catalog.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_dataplex_catalog_list_entries]
+    :end-before: [END howto_operator_dataplex_catalog_list_entries]
+
+.. _howto/operator:DataplexCatalogGetEntryOperator:
+
+Get an Entry
+------------
+
+To retrieve an Entry in specific location in Dataplex Catalog you can
+use :class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCatalogGetEntryOperator`
+
+.. exampleinclude:: /../../providers/tests/system/google/cloud/dataplex/example_dataplex_catalog.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_dataplex_catalog_get_entry]
+    :end-before: [END howto_operator_dataplex_catalog_get_entry]
+
+.. _howto/operator:DataplexCatalogUpdateEntryOperator:
+
+Update an Entry
+---------------
+
+To update an Entry in specific location in Dataplex Catalog you can
+use :class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCatalogUpdateEntryOperator`
+
+.. exampleinclude:: /../../providers/tests/system/google/cloud/dataplex/example_dataplex_catalog.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_dataplex_catalog_update_entry]
+    :end-before: [END howto_operator_dataplex_catalog_update_entry]
+
+.. _howto/operator:DataplexCatalogLookupEntryOperator:
+
+Look up a single Entry
+----------------------
+
+To look up a single Entry by name using the permission on the source system in Dataplex Catalog you can
+use :class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCatalogLookupEntryOperator`
+
+.. exampleinclude:: /../../providers/tests/system/google/cloud/dataplex/example_dataplex_catalog.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_dataplex_catalog_lookup_entry]
```

Comment:
@VladaZakharova could you please add the code example for this operator? I think it was missed, because I do not see it in current PR